packages/contracts-bedrock/src/L1/SimpleStorage.sol

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { Initializable } from "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
```

Comment:
If it helps, we can use solady's initializable (and include it in audit scope), which lets you override the `_initializableSlot()` getter